recipes-private/nsls2ptycho/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set token = environ.get('GH_TOKEN') %}
+
+package:
+  name: nsls2ptycho
+  version: "1.1.1"
+
+source:
+  git_rev: v1.1.1
+  git_url: https://{{ token }}@github.com/leofang/ptycho_gui.git
```

Comment:
This repo seems to be public. Why do we need the token? How is the private repo with the backend (https://github.com/leofang/ptycho) used?

Reply:
Because the private repo is made as a submodule of the public one, see here:
https://github.com/leofang/ptycho_gui/tree/master/nsls2ptycho/core
When building the package, `git clone --recursive` is done so that both repos are accessed, and the token is then used.

Reply:
Oh, right! I remembered it was a submodule, but didn't see it in the root of the repo.